sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md

```diff
@@ -285,14 +292,26 @@ await foreach (Page<BasicDigitalTwin> page in asyncPageableResponseWithCharge.As
 
 ### Build ADT Queries
 
-Build an ADT query using an `AdtQueryBuilder`:
+Build an [ADT query](https://docs.microsoft.com/en-us/azure/digital-twins/concepts-query-language) using an `AdtQueryBuilder`. When using a `Where` clause, conditions are separated from the `Where` keyword (see `whereIsOfModel`)
```

Comment:
Do not include en-us in the link. Reason being is that someone from a different country where locale is not English will still get an English page. If you remove the locale/language in the link, the link will be redirected appropriately to the proper language.

Reply:
good to know!